### ISSUE TITLE
accessibility: Buttons do not have an accessible name

### ISSUE THREAD

**Issue by ychoy** [codeforsanjose/census2020#122]
According to this [audit report from Chrome Dev Tools](https://github.com/codeforsanjose/census2020/files/4054578/Census2020-Audit-Report-0122020.pdf), this can be enhanced to improve accessibility:

When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers. Learn more. https://web.dev/button-name/?utm_source=lighthouse&utm_medium=devtools

Failing Elements
--
```
button<button type="button" class="control-arrow control-prev"></button>
button<button type="button" class="control-arrow control-next"></button>
li<li class="dot selected" value="0" role="button" tabindex="0"></li>
li<li class="dot" value="1" role="button" tabindex="0"></li>
li<li class="dot" value="2" role="button" tabindex="0"></li>
li<li class="dot" value="3" role="button" tabindex="0"></li>
li<li class="dot" value="4" role="button" tabindex="0"></li>
li<li class="dot" value="5" role="button" tabindex="0"></li>
```

**Comment by turnerhayes** [comment]
This is an issue with the react-responsive-carousel library (https://github.com/leandrowd/react-responsive-carousel/issues/266) which seems to be fixed in the latest version, to which we have update in #158
